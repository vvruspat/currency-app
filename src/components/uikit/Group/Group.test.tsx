import { render, screen } from "@testing-library/react";
import { Group } from ".";

test("renders Group", () => {
  render(<Group data-testid="test-group">Test Group</Group>, {});
  const GroupElement = screen.getByTestId("test-group");
  expect(GroupElement).toBeInTheDocument();
  expect(GroupElement).toHaveTextContent("Test Group");
  expect(GroupElement).toMatchSnapshot();
});
