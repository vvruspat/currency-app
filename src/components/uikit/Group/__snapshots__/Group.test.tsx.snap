// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Group 1`] = `
<section
  class="group"
  data-testid="test-group"
>
  Test Group
</section>
`;
